app/kudos/utils.py

```diff
@@ -380,8 +380,12 @@ def _get_contract(self):
             obj: Web3py contract object.
 
         """
-        with open('kudos/Kudos.json') as f:
-            abi = json.load(f)
+        try:
+            with open('kudos/Kudos.json') as f:
+                abi = json.load(f)
+        except:
+            with open('app/kudos/Kudos.json') as f:
```

Comment:
Is this some kind of prod/local difference?

Reply:
no; i think it has to do with different base paths and therfore relative paths from celeryworkers vs gunicorn/manage.py

Reply:
Hmm, the paths should all be the same, but perhaps there is something preventing them or we're not starting from the right context. 

Interesting to note, the failure for the auto merge is because it can't find anything in the app namespace.